Sources/SotoCore/AWSShapes/Payload+async.swift

```diff
@@ -0,0 +1,44 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2021 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if compiler(>=5.5)
+
+import NIOCore
+
+@available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
+extension AWSPayload {
+    /// Construct a stream payload from a `NIOFileHandle`
```

Comment:
Only comment after a quick read through of this is that it's not clear where `NIOFileHandle` comes from or is used in this extension

Reply:
Comment that came from cut and paste. I should have removed. 👍